src/migrations/20180805090233-create-user.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+	up: (queryInterface, Sequelize) => {
+		return queryInterface.createTable('users', {
+			id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+			firstName: {
+				type: Sequelize.STRING
+			},
+			email: {
+				allowNull: false,
+				type: Sequelize.STRING
+			},
+			password_digest: {
+				allowNull: false,
+				type: Sequelize.STRING
+			},
+			isAdmin: {
+				allowNull: false,
+				defaultValue: false,
+				type: Sequelize.BOOLEAN
+			},
```

Comment:
isAdmin?
it looks bad to store it in users table.
it's ok only if there is no more roles will be in your app